src/app/modules/main/wallet_section/send/view.nim

```diff
@@ -172,15 +197,14 @@ QtObject:
   proc transactionWasSent*(self: View, chainId: int, txHash: string, uuid: string, error: string) {.slot} =
     self.transactionSent(chainId, txHash, uuid, error)
 
-  proc authenticateAndTransfer*(self: View, from_addr: string, to_addr: string, tokenSymbol: string,
-    value: string, uuid: string, sendType: int) {.slot.} =
-      self.delegate.authenticateAndTransfer(from_addr, to_addr, tokenSymbol, value, uuid, sendType)
+  proc authenticateAndTransfer*(self: View, value: string, uuid: string) {.slot.} =
+      self.delegate.authenticateAndTransfer(self.selectedSenderAccount.address(), self.selectedRecipient, self.selectedAssetSymbol, value, uuid, self.sendType)
```

Comment:
@Khushboo-dev-cpp in many places in this file I see 4 spaces indentation instead of 2?